### PR TITLE
Design Pattern Strategy Example

### LINKED ISSUE #2: Design Pattern Strategy
Design Pattern Strategy

### PR DESCRIPTION
**Pull request by mertcantahanli**
Fixes #2

### CODE REVIEW

File: src/main/java/org/livecodingturkey/designpatterns/strategy/StrategyMain.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package org.livecodingturkey.designpatterns.strategy;
+
+public class StrategyMain {
+
+    public static void main(String[] args) {
+        Context context = new Context(new ConsecutiveAdd());
```
Comment (huseyinbabal) [inline src/main/java/org/livecodingturkey/designpatterns/strategy/StrategyMain.java:6]: `consecutiveAddContext` would be better

File: src/main/java/org/livecodingturkey/designpatterns/strategy/StrategyMain.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        System.out.println("5 e kadar sayıların toplamı = " + context.executeStrategy(5));
+
+        context = new Context(new Factorial());
```
Comment (huseyinbabal) [inline src/main/java/org/livecodingturkey/designpatterns/strategy/StrategyMain.java:9]: `Context factorialContext = ....`

**Review by huseyinbabal — APPROVED** [review]
LGTM, with minor comment

**Review by huseyinbabal — APPROVED** [review]
LGTM, with minor comment.

File: src/main/java/org/livecodingturkey/designpatterns/strategy/StrategyMain.java (hunk 2 above)
Comment (mertcantahanli) [reply src/main/java/org/livecodingturkey/designpatterns/strategy/StrategyMain.java]: Names changed.

**Pull request merged**